lib/hooks/docker-hub-automated-build/help.html.haml

```diff
@@ -0,0 +1,4 @@
+  %dl
```

Comment:
Thanks!

AFAIK, we can't preview the `help.html.haml` with that.
However, you can confirm the rendered html (hbs) on `app/templates/idobata-hooks/HOOK_NAME/help.hbs` by running `yarn run prepublish`. Unfortunately, probably due to an issue of `hamljs`, it results syntax error on this branch though💦 I'll investigate it later.